index.html

```diff
@@ -0,0 +1,316 @@
+                            </div>
+                        </section>
+                    </div>
+                </section>
+            </div>
+
+            <div>
+
+            </div>
+        </section>
+        
+        <!-- About me Section -->
+        <section id="about">
+            <div id="about-heading">    
+                <h3 id="about-title">About me</h3>
+
+                <p class="about-description"> I can help you build a product , 
+                    feature or website Look through 
+                    some of my work and experience! 
+                    If you like what you see and have 
+                    a project you need coded, 
+                    don't hestiate to contact me.</p>
+                <a href="resume.pdf" download="My Resume"><button id="about-resume-button"><p id="about-resume-button-text">Get My Resume</p></button></a>
+                <img src="img/scattered-backgrounds/about-me-mobile-background1.svg" alt="" class="about-me-mobile1">
+                <img src="img/scattered-backgrounds/about-me-mobile-background2.svg" alt="" class="about-me-mobile2">
+                <img src="img/scattered-backgrounds/bottom-most-mobile-background.svg" alt="" class="bottom-most-mobile-background">
+            </div>
+            
+            <div id="skill-cards">
+                <!-- languages -->
+                <section class="skill-card">
+                    <!-- icon -->
+                    <div class="skill-card-icon">
+                        <div class="skill-card-icon-container"><img class="skill-card-icon-img" src="img/skill/language-img.svg" alt=""></div>
+                    </div>
+
+                    <!-- title and tags -->
+                    <div class="skill-card-content">
+                        <h4 class="skill-card-title">Languages</h4>
+                        <ul class="skill-card-tech">
+                            <li class="skill-card-tech-tag ljavascript"><p class="skill-card-tech-tag-text">Javascript</p></li>
+                            <li class="skill-card-tech-tag lruby"><p class="skill-card-tech-tag-text">Ruby</p></li>
+                            <li class="skill-card-tech-tag lhtml"><p class="skill-card-tech-tag-text">Html</p></li>
+                            <li class="skill-card-tech-tag lcss"><p class="skill-card-tech-tag-text">css</p></li>
+                        </ul>
+                    </div>
+                    
+                </section>
+                <!-- frameworks -->
+                <section class="skill-card">
+                    <!-- icon -->
+                    <div class="skill-card-icon">
+                        <div class="skill-card-icon-container"><img class="skill-card-icon-img" src="img/skill/framework.svg" alt=""></div>
+                    </div>
+                    <!-- title and tags -->
+                    <div class="skill-card-content">
+                        <h4 class="skill-card-title">Frameworks</h4>
```

Comment:
Kindly use an h2 here as it is directly under an h2. it obeys the hx or header hierarchy rule and it is semantic hence will aid user accessibility for screen readers😉

Reply:
Fixed.